titus-server-master/src/main/java/com/netflix/titus/master/mesos/DefaultV3TaskInfoFactory.java

```diff
@@ -230,6 +232,9 @@ public DefaultV3TaskInfoFactory(MasterConfiguration masterConfiguration,
         // Configure IP address allocation
         setSignedAddressAllocation(containerInfoBuilder, task, containerResources);
 
+        // Configure job accepted timestamp
+        setJobAcceptedTimestamp(containerInfoBuilder, job);
```

Comment:
It is not documented here nor in the IDL, why we need this timestamp.

Reply:
We pass the job accept timestamp to agents which then passes it along to Metatron on container launch. Metatron uses this field as part of its evaluation of whether to grant credentials.

This address some issues we've had with long running jobs where the key used to sign the job has since expired and new tasks cannot acquire credentials.